### DISCUSSION TITLE
Dexcom ONE - Companion app and receiver

### DISCUSSION THREAD

**Discussion by Agio56** [NightscoutFoundation/xDrip#2898] [Q&A]
I recently discovered this app (I'm so happy!!). I use a Dexcom ONE system with the official receiver and xDrip+: is it possible to use xDrip and the official Dexcom app AT THE SAME TIME by setting "bluetooth companion"?
I tried but can't connect xDrip+ if the official Dexcom app is already connected.

**Comment by Navid200** [comment]
xDrip gets readings, in Companion app mode, using the notifications provided by the companion app, in this case, the Dexcom One app.
It has been reported that the Dexcom One app may not provide notifications.

There are apps that need a setting to be enabled so that they provide notifications.
Please invesdtigate.

**Reply by Agio56** [reply]
One of the most important features of xDrip+ vs the official app is the ability to calibrate Dexcom ONE. If I use the companion app mode will I still be able to calibrate the system using xDrip+?
Thanks for your patience and understanding, I'm still really a beginner!!

**Reply by Navid200** [reply]
It's the transmitter that must accept the calibration.  And the Dexcom One transmitter accepts calibrations.
Are you sure the Dexcom One app does not allow you to submit calibrations?

No, if you use the Companion app mode, you will not be able to submit calibrations from xDrip because xDrip will not be directly communicating with the transmitter.

**Reply by Agio56** [reply]
Yes. Dexcom ONE app is very minimal. You can only read your glucose and set high and low glucose alerts. Nothing else!!

**Reply by Navid200** [reply]
@Agio56 
Do you still have a question?

**Reply by Agio56** [reply]
Thanks Navid, you were really kind and clear. I will continue to use XDRip, regardless of the official app. I have no more to ask for the moment, XDRip is working very well.
